lib/bugsnag/configuration.rb

```diff
@@ -169,19 +166,26 @@ def clear_request_data
     ##
     # Logs an info level message
     def info(message)
-      logger.info(message)
+      logger.info(format_message(message))
```

Comment:
Use the [block form of Logger logging methods](https://ruby-doc.org/stdlib-1.9.2/libdoc/logger/rdoc/Logger.html#method-i-info) to specify the program name which sent the message. This differentiates Bugsnag log messages from others sent using the same logger.

```diff
- logger.info(format_message(message))
+ logger.info('Bugsnag') { message }
```